setup.sh

```diff
@@ -72,10 +72,12 @@ if [ "${teardown}" != "True" ]; then
         "${moduledir}" | sed -n 's/^Step #2: Service URL: \(.*\)$/\1/p')
 
     echo "Creating trigger subscription for Cloud Run..."
+    # no dead letter topic set; if message fails to process successfully, will continue to try forever
```

Comment:
This is a comment we really want to draw attention to. How about something like

```
# WARNING:  This is set to retry failed deliveries. If there is a bug in main.py this will 
# retry indefinitely, until the message is delete manually.
```

Reply:
Sounds good to me! Thanks for rewording, this message is clear and draws attention